apps/system/js/bluetooth.js

```diff
@@ -1,8 +1,9 @@
 /* global SettingsListener, Service, BluetoothIcon, BluetoothTransferIcon,
           BluetoothHeadphoneIcon, LazyLoader */
-/* exported Bluetooth */
+/* exported Bluetooth1 */
 'use strict';
 
+(function(exports) {
 var Bluetooth = {
```

Comment:
Since the indent fix make followup PR harder to merge, would like to fix indent after related patch landed